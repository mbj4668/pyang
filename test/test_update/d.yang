module d {
  namespace urn:d;
  prefix d;

  import c {
    prefix c;
    revision-date 2014-03-01;
  }

  revision 2014-03-01;

  deviation /c:c/c:q {
    deviate add {
      default 'dummy';
    }
  }
}
