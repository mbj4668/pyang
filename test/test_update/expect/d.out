c@2014-04-01.yang:13: error: CHK_BASE_TYPE_CHANGED
d@2014-04-01.yang:14: error: CHK_DEF_CHANGED
